src/components/ColorPicker/ColorPicker.scss

```diff
@@ -0,0 +1,272 @@
+@import "../../constants/style";
+
+.column__header-color-picker-dropdown {
+  z-index: 220;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  //gap: $spacing--xxs;
+  //padding-top: $spacing--xxs;
+  padding-bottom: $spacing--xxs;
+  box-shadow: 0 0 20px rgba(55, 62, 79, 0.12);
+  //background-color: #ffffff;
+  background-color: $gray--300;
+
+  //width: 40px;
+
+  //box-shadow:
+  //  0 4px 6px -1px rgb(0 0 0 / 0.1),
+  //  0 2px 4px -2px rgb(0 0 0 / 0.1);
+  list-style: none;
+  //padding: 10px;
+  //padding-top: 30px;
+  padding-left: 0;
+  margin-top: auto;
+  border-radius: 24px;
+  //position: inherit;
+  //left: 30px;
+
+  &:hover {
+    background-color: $brighten--slightly;
+  }
+}
+
+.column__header-color-picker-dropdown > li > button {
+  cursor: pointer;
+  border: none;
+  background: none;
+  padding: 0;
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.column__header-color-picker-dropdown > li {
+  all: unset;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  //width: 40px;
+  //height: 40px;
+  cursor: pointer;
+}
+
+.column__header-color-picker-dropdown > ul > li {
+  padding: 0;
+}
+
+[theme="dark"] {
+  .column__header-color-picker-dropdown {
+    background-color: $navy--600;
+  }
+}
+
+.column__header-color-option {
+  width: 23px;
+  height: 23px;
```

Comment:
23 px seems like a odd number 😆 Why not 24?

Reply:
it's 23px in the design :D should I change it to 24?